### PR TITLE
upgrade parent POM and other dependencies

### PR DESCRIPTION
**Pull request by elharo**
@hboutemy

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@ under the License.
   <parent>
     <groupId>org.apache.maven.shared</groupId>
     <artifactId>maven-shared-components</artifactId>
-    <version>31</version>
+    <version>34</version>
```
Comment (hboutemy) [inline pom.xml:28]: please add project.build.outputTimestamp property

Comment (khmarbaise) [reply pom.xml:28]: To have a reproducible build...You can use other plugins etc. as a template.

Comment (elharo) [reply pom.xml:28]: Yes, but what value should this be set to?

Comment (khmarbaise) [reply pom.xml:28]: This is on the other hand a very good question? Maybe Hervé can enlighten us a bit?

**Pull request closed without merging**